sagemaker-training-compiler/huggingface/pytorch_multiple_gpu_multiple_node/language-modeling-multi-gpu-multi-node.ipynb

```diff
@@ -2,15 +2,15 @@
  "cells": [
```

Comment:
<p>Split estimator args into a different cell.</p><p>Add some markdown before the estimator config- "First, we define some basic parameters common to all estimators.</p><p><strong>Note</strong>: We recommend you to turn the SageMaker Debugger's profiling and debugging tools off to avoid additional overheads."</p><p>Check https://github.com/aws/amazon-sagemaker-examples/blob/main/sagemaker-training-compiler/huggingface/pytorch_multiple_gpu_single_node/language-modeling/gpt-2.ipynb</p><p> </p><p>Do the same for the common hyperparameters as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>